### PR TITLE
reintroduce hiding of windows for sectorview

### LINKED ISSUE #4298: sectorview new gui no longer supports tab
old gui supported tab to hide forms and info, new gui does not

### PR DESCRIPTION
**Pull request by richardpl**
Closes #4298.

### CODE REVIEW

File: data/pigui/modules/hyperjump-planner.lua [hunk 1]
```diff
@@ -234,7 +235,12 @@ local function displayHyperJumpPlanner()
...
         hyperjump_route = Engine.SectorMapGetRoute()
         route_jumps = Engine.SectorMapGetRouteSize()
-        showHyperJumpPlannerWindow()
+	if ui.isKeyReleased(ui.keys.tab) then
+		hideHyperJumpPlaner = not hideHyperJumpPlaner;
+	end
+	if not hideHyperJumpPlaner then
+		showHyperJumpPlannerWindow()
+	end
     end
```
Comment (impaktor) [inline data/pigui/modules/hyperjump-planner.lua]: Indentation looks wrong.

**Comment by impaktor** [comment]
I assume Lua-guys @laarmen and @ecraven share the sentiment that indentation should be consistent in a file. (It seems like this file used spaces, which was less good, as we indent Lua wit tabs.) 

@richardpl changing indentation is super easy, unless you're using notepad or something strange as IDE. Emacs, Vim, sublime, atom, or whatever you use as IDE have indentation correction at a single key stroke, or similar. 

Alternative, I can push a fix to this branch.

**Comment by impaktor** [comment]
I can push a commit fixing the indentation tomorrow.

**Comment by laarmen** [comment]
Actually I usually moan about indentation more because of diff readability
and because I know you (impaktor) like it when we're consistent about
things. All together I don't care particularly about it

**Pull request closed without merging**